ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.model.ts

```diff
@@ -3,17 +3,18 @@ export class SequencingOrder {
 
   constructor( public sampleType: string, public sample: string, public sampleStatus: string, public collectionDate: string,
                public sequencingOrderDate: string, public tissueId: string, public dsmKitRequestId: string,
-               public sequencingRestriction: string, public lastStatus: string, public lastOrderNumber: string ) {
+               public sequencingRestriction: string, public lastStatus: string, public lastOrderNumber: string,
+               public pdoOrderId: string ) {
 
   }
 
   public static parse( json ): SequencingOrder {
     return new SequencingOrder( json.sampleType, json.sample, json.sampleStatus, json.collectionDate, json.sequencingOrderDate,
-      json.tissueId, json.dsmKitRequestId, json.sequencingRestriction, json.lastStatus, json.lastOrderNumber );
+      json.tissueId, json.dsmKitRequestId, json.sequencingRestriction, json.lastStatus, json.lastOrderNumber, json.pdoOrderId );
   }
 
   public static getEmptyInstance(): SequencingOrder{
-    return new SequencingOrder( '', '', '', '', '', '', '', '', '', '' );
+    return new SequencingOrder( '', '', '', '', '', '', '', '', '', '', '' );
```

Comment:
**_just a tip fyi:_**

In this particular case:
the arguments (11 empty spaces) can be simplified as to not miss something
like ` return new SequencingOrder(  ...Array(11).fill('') );`

Another option is to have default values in the class constructor:

```
constructor( 
  public sampleType: string = '', public sample: string = '', public sampleStatus: string ='', 
  public collectionDate: string = '', public sequencingOrderDate: string = '', public tissueId: string ='',
  public dsmKitRequestId: string = '', public sequencingRestriction: string ='', public lastStatus: string ='', 
  public lastOrderNumber: string ='' , public pdoOrderId: string = ''
) { }

public static getEmptyInstance(): SequencingOrder{
    return new SequencingOrder(); // default values will be used
}
```

Reply:
Or you could just use static factory methods like it's mentioned in this [article](https://medium.com/young-coder/supporting-multiple-constructors-in-javascript-48cab12264ed) (As far as I know, we are already using that approach in several classes)

Reply:
I applied the `return new SequencingOrder( ...Array(11).fill('') );` because I am not sure if I want it as a default... I'd rather they default to undefined...